.github/workflows/run-tests.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os:               [ubuntu-latest, windows-latest]
-        python-version:   ['2.7', '3.10', 'pypy2.7', 'pypy3.8']
+        python-version:   ['2.7', '3.x', 'pypy2.7', 'pypy3.x']
```

Comment:
For some reason, the GitHub action for setting up python does not resolve ".x" for pypy versions.